spec/cypress/support/step_definitions/common-navigation.js

```diff
@@ -22,8 +22,10 @@ const pagePaths = {
   privacy: "/privacy-policy",
   accessibility: "/accessibility-statement",
   dashboard: "/dashboard",
+  "Manage your training":
+    "/schools/:id",
   "2021 cohort CIP materials info":
-    "/schools/:id/cohorts/2021/core-programme/materials/info",
+    "/schools/:id/cohorts/2021/programme-choice",
```

Comment:
Something here is almost certainly what is breaking the linter. Line length most likely.

If you run
```
yarn run prettier spec/cypress/support/step_definitions/common-navigation.js --write
```
the file should get 'fixed' for you.

Reply:
That worked, thanks!